apps/aechannel/src/aesc_fsm.erl

```diff
@@ -461,6 +484,30 @@ strict_checks(Fsm, Strict) when is_boolean(Strict) ->
     gen_statem:call(Fsm, {strict_checks, Strict}).
 -endif.
 
+%% ======================================================================
```

Comment:
These functions should be moved to another module to not extend the complexity of `aesc_fsm`. I'd suggest `aesc_fsm_stats`.